frontend/mono-ui/web/rainmaker/dev-packages/egov-pt-dev/src/ui-config/screens/specs/pt-mutation/iFunctions.js

```diff
@@ -308,14 +325,13 @@ let tenantUniqueId = filterTenant && filterTenant[0] && filterTenant[0].city &&
       {}
     );
       
-
       // const response = searchSampleResponse();
-      let propertyData = finalResponse.Properties.map(item => ({
+      let propertyData = await finalResponse.Properties.map((item) => ({
+        
         ["PT_COMMON_TABLE_COL_PT_ID"]:
           item.propertyId || "-",
-        ["PT_COMMON_TABLE_COL_OWNER_NAME"]: item.owners[getIndexofActive(item)] && item.owners[getIndexofActive(item)].name || "-",
-        ["PT_GUARDIAN_NAME"]:
-        item.owners[getIndexofActive(item)] && item.owners[getIndexofActive(item)].fatherOrHusbandName || "-",
+        ["PT_COMMON_TABLE_COL_OWNER_NAME"]: getName(item),
```

Comment:
This is won't work. We have to get only active user and need to show. If the property went gone through then multiple users will be there and we have pick active user then need to show that in the search response.